contratospr/contracts/views.py

```diff
@@ -191,6 +197,144 @@ def contractors(request):
     return render(request, "contracts/contractors.html", context)
 
 
+def get_trend(fiscal_year):
+    start_date = timezone.make_aware(datetime.datetime(fiscal_year, 7, 1))
```

Comment:
@froi can we also change this to use `contratospr.contracts.utils.get_fiscal_year_range`

https://github.com/TheIndexingProject/contratospr-api/blob/2c61ee918724d74532c2df05624d641de77c4515/contratospr/contracts/utils.py#L19

Reply:
Creo que `get_fiscal_year_range` esta incorrecto.

Como esta ahora mismo, si paso 2018 lo que devuelve es
- start_date = 2018-07-01
- end_date = 2019-06-30

Estos valores son el año fiscal 2019. El año fiscal 2018 es 2017-07-01 al 2018-06-30.

Voy a hacer este cambio de una vez.